src/visualization/recipes_makie.jl

```diff
@@ -108,7 +108,9 @@ iplot(u, semi; kwargs...) = iplot(wrap_array_native(u, semi), mesh_equations_sol
 # This initializes a Makie recipe, which creates a new type definition which Makie uses to create
 # custom `trixiheatmap` plots. See also https://makie.juliaplots.org/stable/recipes.html
 @Makie.recipe(TrixiHeatmap, plot_data_series) do scene
-  Makie.Theme(;)
+  Makie.Theme(
+    colormap = :thermal,
+  )
```

Comment:
A related question that comes to my mind here (since I'm testing it): Is it possible to add the colorbars to the Makie plots as we do for the Plots plots? For example, `GLMakie.plot(sol, plot_mesh=false)` gives me the plot below, which is hard to interpret without colorbars.

![image](https://user-images.githubusercontent.com/12693098/130495161-8c8391c6-74c7-4bd7-a776-6b9f0ffac181.png)

Reply:
Yeah, let me give that a shot. I tried before and it looked odd, but I think I have to do something with Makie's "Aspect" tools.

Reply:
If it's not easy to do right now, please open an issue for it

Reply:
Sounds good.